index.js

```diff
@@ -162,6 +164,58 @@ const inventoryCommand = (message) => {
   });
 };
 
+const lookUpCommand = (message) => {
+  const originalMessage = message.content;
+  const fuzzySearchKey = originalMessage.split(" ").splice(1).join(" ");
+  const fuzzySearchUrl =
+    "https://db.ygoprodeck.com/api/v7/cardinfo.php?&fname=";
+
+  console.log(fuzzySearchUrl + fuzzySearchKey);
+
+  axios
+    .get(fuzzySearchUrl + fuzzySearchKey)
+    .then((cardSearchResults) => {
+      const objectsReturnedBySearch = cardSearchResults.data.data;
+      const exactMatchObject = objectsReturnedBySearch.find(
```

Comment:
similarly, we know the object is a card, so we can replace the `Object` with `Card`